libvmaf/src/libvmaf.c

```diff
@@ -42,13 +42,24 @@
 #include "vcs_version.h"
 
 #ifdef HAVE_CUDA
-#include "libvmaf/vmaf_cuda.h"
-#include "libvmaf/vmaf_cuda_state.h"
+#include "libvmaf/libvmaf_cuda.h"
 
 #include "cuda/common.h"
 #include "cuda/cuda_helper.cuh"
 #include "cuda/picture_cuda.h"
 #include "cuda/ring_buffer.h"
+
+typedef struct VmafCudaConfiguration3 {
+    struct {
+        unsigned w, h;
+        unsigned bpc;
+        enum VmafPixelFormat pix_fmt;
+    } pic_params;
+    enum VmafCudaPicturePreallocationMethod pic_prealloc_method;
+    int device_id;
+    int stream_priority;
+} VmafCudaConfiguration3;
+//^FIXME
```

Comment:
Due to the FIXME  i am guessing this 3 has to go away ?

Reply:
Yes, thanks. Meant to clean this up.